django/views/templates/default_urlconf.html

```diff
@@ -1,44 +1,487 @@
+            -webkit-text-decoration-skip: objects;
+            color: #20AA76;
+            text-decoration: none
+          }
+
+          img {
+            border-style: none
+          }
+
+          .next-step, header {
+            border-bottom: 1px solid #efefef
+          }
+
+          svg:not(:root) {
+            overflow: hidden
+          }
+
+          body {
+            max-width: 960px;
+            color: #525252;
+            font-family: Roboto, sans-serif
+          }
+
+          main {
+            text-align: center
+          }
+
+          .u-clearfix:after {
+            content: "";
+            clear: both;
+            display: table
+          }
+
+          .exhaust.two, .fuel, .removehole {
+            display: none
+          }
+
+          h1, h2, h3, h4, h5, p, ul {
+            padding: 0;
+            margin: 0;
+            font-weight: 400
+          }
+
+          header {
+            padding-top: 20px;
+            padding-bottom: 10px
+          }
+
+          .logo {
+            float: left
+          }
+
+          .logo h2 {
+            font-weight: 700;
+            margin-top: 0px
+          }
+
+          .release-notes {
+            float: right;
+            margin-top: 7px
+          }
+
+          .release-notes p {
+            font-size: 14px
+          }
+
+          .figure {
+            margin-top: 19vh;
+            height: 173px
+          }
+
+          .figure__animation {
+            max-width: 265px;
+            position: relative;
+            z-index: -9
+          }
+
+          .tip {
+            stroke-width: 3px;
+            -webkit-transform: translateY(-7px);
+            transform: translateY(-7px)
+          }
+
+          .body {
+            fill: #6DDCBD;
+            -webkit-transform: scaleX(1.25);
+            transform: scaleX(1.25);
+            -webkit-transform-origin: center;
+            transform-origin: center;
+            stroke-width: 3px
+          }
+
+          .box {
+            position: absolute;
+            bottom: 0;
+            left: 0
+          }
+
+          .circle {
+            fill: #6DDCBD;
+            stroke: #fff;
+            stroke-width: 7px
+          }
+
+          .exhaust {
+            border-bottom-left-radius: 4px
+          }
+
+          .exhaust.two, .exhaust__line {
+            -webkit-animation: thurst 70ms infinite ease-in-out alternate;
+            animation: thurst 70ms infinite ease-in-out alternate
+          }
+
+          .smoke {
+            -webkit-animation: smoke .1s infinite ease-in-out alternate;
+            animation: smoke .1s infinite ease-in-out alternate;
+            -webkit-transform-origin: center;
+            transform-origin: center;
+            opacity: .8
+          }
+
+          @-moz-document url-prefix() {
+            .smoke {
+              animation: smokeFirefox .2s infinite ease-in-out alternate
+            }
+          }
+
+          @-webkit-keyframes smoke {
```

Comment:
Is there any problem with `@-webkit-keyframes smoke, @keyframes smoke { ...` rather than repeating the rule?